gateway/src/apicast/version.lua

```diff
@@ -1 +1 @@
-return "3.6.0"
+return "3.9.0"
```

Comment:
this should have an alpha or beta suffix, right?

Reply:
I think that we always keep it as 0. but let me change.